components/Header/Header.tsx

```diff
@@ -65,29 +65,33 @@ export const Header = ({ jobsAvailable = 0 }: HeaderProps) => {
 
     return (
         <div className={styles.container}>
-            <a
-                className={styles.torchboxLogo}
-                href="https://torchbox.com"
-                aria-label="Home"
-            >
-                <TorchboxLogo />
-            </a>
-            <MobileMenuButton
-                isOpen={isOpen}
-                toggleMobileMenu={toggleMobileMenu}
-            />
-            <MobileNav
-                isOpen={isOpen}
-                links={NAVIGATION_LINKS}
-                navMenuRef={mobileNavRef}
-                jobsAvailable={jobsAvailable}
-            />
-            <DesktopNav
-                isOpen={isOpen}
-                links={NAVIGATION_LINKS}
-                toggleMenu={toggleMobileMenu}
-                jobsAvailable={jobsAvailable}
-            />
+            <div className={styles.navigationContainer}>
+                <div className={styles.logo}>
+                    <a
+                        className={styles.logoLink}
+                        href="https://torchbox.com"
```

Comment:
**Nickpick** **non-blocking**: Ideally this would be a global variable referenced by an attribute in a urls object.

Reply:
Added this as a ticket for refactoring, this would be great to set up sitewide